Sources/ComposableArchitecture/Store.swift

```diff
@@ -327,7 +330,7 @@ public final class Store<State, Action> {
     state toLocalState: @escaping (State) -> LocalState,
     action fromLocalAction: @escaping (LocalAction) -> Action
   ) -> Store<LocalState, LocalAction> {
-    self.threadCheck(status: .scope)
+    self.performChecks(event: .scope)
     var isSending = false
     let localStore = Store<LocalState, LocalAction>(
```

Comment:
If we only want the root store to emit, we can flip `stackChecksEnabled` to `false` for these instances.